test/collections_test_helper.rb

```diff
@@ -0,0 +1,51 @@
+# rubocop:disable Metrics/LineLength
+require_relative "./test_helper"
+
+VALID_COLLECTION_METADATA_KEYS = %w[collection created_by display_name image items].freeze
+
+REQUIRED_COLLECTION_METADATA_KEYS = %w[display_name].freeze
+
+MAX_COLLECTION_LENGTH = 40
```

Comment:
Maybe `MAX_COLLECTION_SLUG_LENGTH` so it's clear that's how long the `collection:` key that represents the URL slug can be, as opposed to how many items are allowed in a collection.